### PR TITLE
Change the solution2 to also use 'variables'

### PR DESCRIPTION
**Pull request by dantxal**
# Summary

This change would add the use of variables to the solution as required by the exercise.

## Test Plan
1. `yarn copy-env`
2. `yarn relay`
3. `yarn start`

We will see 5 cards instead of 10, as defined in the variable `{ first: 5 }`.

### What's required for testing (prerequisites)?
No additional prerequisites
### What are the steps to reproduce (after prerequisites)?
No additional steps

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Review by sibelius — APPROVED** [review]

File: solutions/02-useLazyLoadQuery/src/App.tsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ import { AppQuery } from './__generated__/AppQuery.graphql';
 const App = () => {
   const response = useLazyLoadQuery<AppQuery>(
     graphql`
-      query AppQuery {
-        posts(first: 10) {
+      query AppQuery ($first: Number) {
+        posts(first: $first) {
```
Comment (AugustoCalaca) [inline solutions/02-useLazyLoadQuery/src/App.tsx:13]: also try using a `defaultValue`

**Review by AugustoCalaca — APPROVED** [review]

File: solutions/02-useLazyLoadQuery/src/App.tsx [hunk 2]
```diff
@@ -9,8 +9,8 @@ import { AppQuery } from './__generated__/AppQuery.graphql';
 const App = () => {
   const response = useLazyLoadQuery<AppQuery>(
     graphql`
-      query AppQuery {
-        posts(first: 10) {
+      query AppQuery ($first: Number) {
```
Comment (sibelius) [inline solutions/02-useLazyLoadQuery/src/App.tsx:12]: Number does not exist

image

**Review by sibelius — CHANGES_REQUESTED** [review]
Number does not exist

**Review by sibelius — APPROVED** [review]
nice

File: solutions/02-useLazyLoadQuery/src/App.tsx (hunk 1 above)
Comment (dantxal) [reply solutions/02-useLazyLoadQuery/src/App.tsx]: Correct me if I'm wrong, but defaultValue is only available inside '@argumentDefinitions'

**Pull request merged**